### DISCUSSION TITLE
Highlighting HBS button depending on which pop-up is active?

### DISCUSSION THREAD

**Discussion by pgale** [Clooos/Bubble-Card#659] [General]
Not sure if this should already happen or it's not showing due to my theme. What I'd like to see is the horizontal button stack button highlighted (different colour background would do it) when the popup it controls is active. Kind of an obvious one I think. 

Is that already core functionality or would I need to do that with open/close actions on each button on each dashboard?

**Comment by pgale** [comment]
Actually, is `highlight_current_view` meant to do something like that? It does nothing I can see on my dashboards.

**Reply by Clooos** [reply]
Hi, yes this should works with it, there is a slight opacity change on the active HBS button when the pop-up is open. There is also a CSS class that is added (but I don't remember its exact name), so you can tweak that with some custom styles if you want to.
